client/me/purchases/concierge-banner/index.jsx

```diff
@@ -2,24 +2,27 @@ import { Card } from '@automattic/components';
 import { localize } from 'i18n-calypso';
 import PropTypes from 'prop-types';
 import { Component } from 'react';
+import { connect } from 'react-redux';
 import { Banner } from 'calypso/components/banner';
 import TrackComponentView from 'calypso/lib/analytics/track-component-view';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 import {
-	CONCIERGE_HAS_AVAILABLE_INCLUDED_SESSION,
-	CONCIERGE_HAS_AVAILABLE_PURCHASED_SESSION,
+	CONCIERGE_HAS_AVAILABLE_SESSION,
 	CONCIERGE_HAS_UPCOMING_APPOINTMENT,
 } from 'calypso/me/concierge/constants';
+import getCurrentRoute from 'calypso/state/selectors/get-current-route';
+import getCurrentRouteParameterized from 'calypso/state/selectors/get-current-route-parameterized';
+import { getSiteSlug } from 'calypso/state/sites/selectors';
+
 import './style.scss';
 
 class ConciergeBanner extends Component {
 	static propTypes = {
 		bannerType: PropTypes.oneOf( [
 			CONCIERGE_HAS_UPCOMING_APPOINTMENT,
-			CONCIERGE_HAS_AVAILABLE_INCLUDED_SESSION,
-			CONCIERGE_HAS_AVAILABLE_PURCHASED_SESSION,
+			CONCIERGE_HAS_AVAILABLE_SESSION,
 		] ).isRequired,
-		sites: PropTypes.array.isRequired,
+		availableSessions: PropTypes.array.isRequired,
```

Comment:
Nit: it would also be good to document the `showPlaceholder`, `nextAppointmentSiteId`, and `siteId` props too.

Reply:
I've done that one too.